src/index.js

```diff
@@ -1,100 +1,48 @@
 import './styles/main.scss';
+import {UI} from'./modules/api_movies.js';
+import {addLikes} from'./modules/api_likes.js';
 
-class UI {
-  static getMovies = async () => {
-    const res = await fetch('https://api.tvmaze.com/shows');
-    const data = await res.json();
 
-    const likeres = await fetch(
-      'https://us-central1-involvement-api.cloudfunctions.net/capstoneApi/apps/w6gyfRjKef7dpeJ8lwcd/likes'
-    );
-    const likedata = await likeres.json();
-    console.log(likedata);
+//-----Likes----
+document.addEventListener('click', (e) => {
+ //e.preventDefault();
+if(e.target.matches('.like')){
+  console.log(e.target.id, 'soy id');
```

Comment:
Please, remove console.log() functions from your codes before pushing them online so we can have a lighter application.